pom.xml

```diff
@@ -292,22 +287,54 @@
                 </configuration>
             </plugin>
 
-            <plugin>
-                <groupId>com.diffplug.spotless</groupId>
-                <artifactId>spotless-maven-plugin</artifactId>
+            <!-- <plugin>
+                <groupId>org.openrewrite.maven</groupId>
+                <artifactId>rewrite-maven-plugin</artifactId>
+                <version>6.18.0</version>
+                <configuration>
+                    <activeRecipes>
+                        <recipe>org.openrewrite.staticanalysis.CodeCleanup</recipe>
+                    </activeRecipes>
+                    <checkstyleConfigFile>${maven.multiModuleProjectDirectory}/checkstyle.xml</checkstyleConfigFile>
+                </configuration>
                 <executions>
                     <execution>
+                        <id>validate</id>
                         <phase>validate</phase>
                         <goals>
-                            <goal>apply</goal>
+                            <goal>run</goal>
                         </goals>
                     </execution>
                 </executions>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.openrewrite.recipe</groupId>
+                        <artifactId>rewrite-static-analysis</artifactId>
+                        <version>2.17.0</version>
+                    </dependency>
+                </dependencies>
+            </plugin> -->
+
+            <plugin>
```

Comment:
This I will remove from here but wanted to leave it here for a bit to show what I used.

Reply:
Again, we could keep it so that we can manually run it when we need it.